Userland/Libraries/LibGUI/FilteringProxyModel.cpp

```diff
@@ -94,7 +94,7 @@ void FilteringProxyModel::filter()
 
```

Comment:
From the contribution guidelines, [here](https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#code-submission-policy):

>Don't use specific component names, e.g. C++ class names, as the category either - mention them in the summary instead. E.g. LibGUI: Brief description of what's being changed in FooWidget rather than FooWidget: Brief description of what's being changed

Also, please don't use abbreviation [CodingStyle.md](https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#right-1)

Reply:
You're technically not wrong but we just (almost) don't use abbreviation at all even if the documentation isn't explicit about commit messages.

Class names are not authorized as a category, you can include it in the message. This would look something like:
`LibGUI: Display commands on CommandPalette's start up`

Reply:
Done as you suggested, I included FilteringProxyModel's name in the message instead in the title.

Reply:
(Just to let you know, you're the only one able to hit the "resolve conversation" button)